test/mocha-utils.js

```diff
@@ -92,6 +93,16 @@ global.itFailsFirefox = (...args) => {
     return it(...args);
 };
 
+global.itFailsWindowsUntilDate = (date, ...args) => {
+  const today = new Date();
```

Comment:
Since we only need the timestamp to compare:

```suggestion
  const today = Date.now;
```

Since this is pretty short we don't really need the `today` variable (better: `now`?) anymore.

Reply:
We could make this function accept a string for the `date` parameter, since all callers do `new Date(string)` anyway.

Reply:
i.e. `itFailsWindowsUntilDate('2020-06-01', blah)`

Reply:
May do that in a follow up PR - I'm also tempted to tidy up the test helpers a bit. Right now we have `itFailsFirefox` (with no date) and `itFailsWindowsUntilDate`. I'd quite like a `itFails({ product: 'firefox', date: '... })` or similar API rather than bespoke methods every time